### PR TITLE
Move GitHub corner and visually-hidden class

### LINKED ISSUE #1295: Incorrect Alignment of github corners in codesandbox preview
In the below GIF the github corner can be seen to be incorrectly aligned on initial loading. I cannot however this issue on excalidraw.com, when the device width is smaller.

Excalidraw-github-corner-issue

Steps to reproduce:
1) Open codesandbox url mentioned in the readme.
2) Reload the codesandbox preview on the right side.

### PR DESCRIPTION
**Pull request by yongdamsh**
Resolves #1295 

github_corner

### CODE REVIEW

**Comment by dwelle** [comment]
I missed when `visually-hidden` was added. What's the point of that? AFAIK, google penalizes these hacks.

**Comment by dwelle** [comment]
E.g. here: https://support.google.com/webmasters/answer/66353?hl=en

> Hiding text or links in your content to manipulate Google’s search rankings can be seen as deceptive and is a violation of Google’s Webmaster Guidelines. 

But they also say:

> However, not all hidden text is considered deceptive. For example, if your site includes technologies that search engines have difficulty accessing, like JavaScript, images, or Flash files, using descriptive text for these items can improve the accessibility of your site.

So hard to day. That page is also arguable old, and 

> technologies that search engines have difficulty accessing, like JavaScript, images, or Flash files,

no longer applies.

From various places I've just heard that hiding elements to hack SEO is a bad idea.

**Comment by yongdamsh** [comment]
I don't know the details, but it started with #534, and it was done to improve accessibility. I think it's okay to keep it. Whether to delete visually hidden text will follow the discussion.

**Comment by dwelle** [comment]
Yea, it's unrelated to this PR, but didn't want to create a new issue.

We can merge this.

(Btw, is there a perf benefit to memoizing the GH component that has no props/state instead of rerendering it every time as we do now?)

File: src/components/GitHubCorner.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+export const GitHubCorner = React.memo(() => {
+  // https://github.com/tholman/github-corners
+  return (
```
Comment (lipis) [inline src/components/GitHubCorner.tsx:5]: we can get rid of this return if we put the comment above the signature.

**Pull request merged**